src/test/scala/LiveVarsAnalysisTests.scala

```diff
@@ -88,9 +88,9 @@ class LiveVarsAnalysisTests extends AnyFunSuite, TestUtil {
     val liveVarAnalysisResults = InterLiveVarsAnalysis(program).analyze()
 
     val procs = program.procs
-    assert(liveVarAnalysisResults(procs("main")) == Map(R30 -> TwoElementTop))
-    assert(liveVarAnalysisResults(procs("callee1")) == Map(R0 -> TwoElementTop, R30 -> TwoElementTop))
-    assert(liveVarAnalysisResults(procs("callee2")) == Map(R1 -> TwoElementTop, R30 -> TwoElementTop))
+    // assert(liveVarAnalysisResults(procs("main")) == Map(R30 -> TwoElementTop))
+    assert(liveVarAnalysisResults(procs("callee1")) == Map(R0 -> TwoElementTop))
+    assert(liveVarAnalysisResults(procs("callee2")) == Map(R1 -> TwoElementTop))
   }
```

Comment:
Do we want to remove R30 from these? It makes sense to still consider it 'live' since it is called in a Return?